OMGVA PoS/Data layer/Models/Order.cs

```diff
@@ -9,7 +9,13 @@ public class Order
         public decimal Tip { get; set; }
         public string RefundReason { get; set; }
         public long UserId { get; set; }
-        public long PaymentId { get; set; }
```

Comment:
Is there a reason why you specifically deleted this id?

Reply:
For split payments, reverted now